iocs/tlbc2IOC/iocBoot/ioctlbc2/autosave.cmd

```diff
@@ -11,6 +11,8 @@ save_restoreSet_SeqPeriodInSeconds(300)
 
 set_savefile_path("C:/ProgramData/ADTLBC2", "autosave")
 
+dbLoadRecords("db/save_restoreStatus.db", "P=${P}${R}")
+
```

Comment:
This won't work. We are using `PREFIX` for our PV prefix, not `P` or `R`; and `AUTOSAVE_PREFIX` is what's being passed to `save_restoreSet_status_prefix`. Everything has to be fixed to use `"$PREFIX"`.

Reply:
This went the opposite direction, I was suggesting using `PREFIX` here and for the `save_restoreSet_status_prefix` function, and leaving `AUTOSAVE_PREFIX` only for files.